packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -514,10 +514,11 @@ class Auth {
      * @returns {Promise}
      */
     createOCAPISession() {
+        // Example URL 'https://test.phased-launch-testing.com/on/demandware.store/Sites-RefArch-Site/en_US/SLASSessionHelper-SyncHybridSession'
         return fetch(this.OCAPISessionsURL, {
-            method: 'POST',
+            method: 'GET',
             headers: {
-                Authorization: 'Bearer ' + this.get('access_token')
+                Authorization: this.get('access_token')
```

Comment:
It looks like the stored value doesn't include `Bearer `, so why was that removed from the header?

Reply:
The controller in plugin SLAS appends the `Bearer` prefix before making the call to the `/sessions` endpoint in a helper method